edit-post/hooks/more-menu/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import CopyContentMenuItem from '../../components/header/copy-content-menu-item';
```

Comment:
You're hooking CopyContentMenuItem into Gutenberg but keeping its definition outside `edit-post/hooks`. That's interesting; could you explain why?

Reply:
I wasn't sure where to put it. I guess you are right that it should stay as subfolder inside `edit-post/hooks/` :)